### PR TITLE
Additional clarifications

### PR DESCRIPTION
**Pull request by ved-rivos**

### CODE REVIEW

File: cfi_backward.adoc [hunk 1]
```diff
@@ -228,6 +228,10 @@ else
 endif
 ----
 
+The `ssp` is incremented by `sspop` only if the load from shadow stack completes
```
Comment (deepak0414) [inline cfi_backward.adoc:231]: `sspopchkra` need similar wordings?

The `ssp` is incremented by `sspopchkra` only if the load from shadow stack completes successfully.

Comment (ved-rivos) [reply cfi_backward.adoc:231]: Please see PR #59

Comment (deepak0414) [reply cfi_backward.adoc:231]: great

**Review by deepak0414 — APPROVED** [review]

File: cfi_forward.adoc [hunk 2]
```diff
@@ -377,5 +377,6 @@ current `ELP` and `ELP` is set to `NO_LP_EXPECTED`.
 
 `MRET` or `SRET` instruction is used to return from a trap in M-mode or S-mode
 respectively. When executing an `xRET` instruction, the `ELP` is set to `xPELP`
-and xPELP is set to `NO_LP_EXPECTED`. The trap handler should restore the
-preserved `lpl` value before invoking `SRET` or `MRET`.
+if `xFCFIE` is 1 at privilege level. The `xPELP` is set to `NO_LP_EXPECTED`.
```
Comment (kwalker27) [inline cfi_forward.adoc:380]: Maybe say at the targeted privilege level? It seems a little vague as is.

File: cfi_backward.adoc [hunk 3]
```diff
@@ -205,8 +208,8 @@ The operation of the `sspush` instructions is as follows:
 [source, text]
 ----
 If (xBCFIE = 1)
+   *[ssp] = [src - (XLEN/8)]   # Store src value to ssp - XLEN/8
```
Comment (kwalker27) [inline cfi_backward.adoc:211]:
```suggestion
   *[ssp - (XLEN/8)] = [src]   # Store src value to ssp - XLEN/8
```

**Review by kwalker27 — CHANGES_REQUESTED** [review]
Thanks for the updates

File: cfi_forward.adoc (hunk 2 above)
Comment (ved-rivos) [reply cfi_forward.adoc]: agree

File: cfi_backward.adoc (hunk 3 above)
Comment (ved-rivos) [reply cfi_backward.adoc]: fixed.

**Pull request merged**